CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # @digitalbazaar/mocha-w3c-interop-reporter ChangeLog
 
+## 1.x.x - 2024-mm-dd
+
+### Added
+- Tests can now provide `this.test.link` to link test names to a destination
```

Comment:
where is the code for this feature or was updating the changelog missed?

Reply:
@aljones15 I missed updating the changelog in #37.